duckduckgo/Android
# Rename site-permissions camera icon

Task/Issue URL: https://app.asana.com/0/488551667048375/1206057713910055/f

### Description
Renamed camera icon in site-permissions module for the updated name (video-24)

### Steps to test this PR _(Optional)_

- Install from branch
- Go to Settings>Permissions>Site Permissions
- [ ] Check icon for camera option is correct

### No UI changes
